validator/src/main.rs

```diff
@@ -1543,6 +1543,12 @@ pub fn main() {
                 .default_value(default_rpc_max_request_body_size)
                 .help("The maximum request body size accepted by rpc service"),
         )
+        .arg(
+            Arg::with_name("rpc_prometheus_metrics")
+                .long("rpc-prometheus-metrics")
```

Comment:
Yeah... if we don't want to tie it to the `rpc` name, it's better to omit this prefix.
IMO `--enable-prometheus-metrics` sounds better.

Anyway, one of the reasons I've added the `rpc` prefix is because the config parameter goes to the Rpc service config, and also because the endpoint is on the RPC port.

Reply:
changed to `enable-prometheus-metrics`